docker/development/docker-compose.yml

```diff
@@ -100,8 +100,8 @@ services:
       URL_HOST: localhost
       URL_HOST_SHORT: localhost
       SECRET_KEY_BASE: testenvironment
-      ERDBEERE_SERVER: https://erdbeere.mathi.uni-heidelberg.de
-      ERDBEERE_API: https://erdbeere.mathi.uni-heidelberg.de/api/v1
+      ERDBEERE_SERVER: http://erdbeere:3005
+      ERDBEERE_API: http://erdbeere:3005/api/v1
```

Comment:
haha nice that our local setup doesn't need to call the production environment anymore ;)